product-service/handlers/getProductsById.js

```diff
@@ -0,0 +1,21 @@
+import productList from './productList.json';
+import { headers } from '../constants/headers';
+
+export const getProductsById = async event => {
+  const { productId } = event.pathParameters;
+  try {
```

Comment:
I'd suggest moving duplicate logic (like error handling, response formatting) into separate `lib/` functions and use them across the handlers.

It's a wish for the future, no need to implement it right now. It won't affect your score.

Reply:
changed it